demisto_sdk/common/hook_validations/release_notes.py

```diff
@@ -72,45 +65,44 @@ def is_release_notes_changed(self):
                 return True
 
         print_error(F'File {self.release_notes_path} is not formatted according to '
-                    F'release notes standards.\nFix according to {release_notes_standard}')
+                    F'release notes standards.\nFix according to {self.RELEASE_NOTES_STANDARD}')
         return False
 
     def is_valid_release_notes_structure(self):
-        # type: () -> bool
         """Validates that the release notes written in the correct manner.
 
         one_line_release_notes_regex meaning:
-            dont start with any of the characters in the first []
             starts with a letter or number
             ends with '.'
 
         multi_line_release_notes_regex meaning:
-            starts with tab, then '-' ,then space
-            ends with '.'
+            Option #1:
+                starts with tab or two to four spaces, then '-' ,then space
+                each line ends with '.'
+            Option #2:
+                starts with '-' with none or one trailing spaces.
 
         Returns:
             bool. True if release notes structure valid, False otherwise
-
         """
         hyphen_release_notes_regex = r'- ?'
         one_line_release_notes_regex = r'[a-zA-Z0-9].*\.$'
         multi_line_release_notes_regex = r'(\t+| {2,4})- .*\.$|- ?$'
 
-        release_notes_comments = self.release_notes.split('\n')
-        print(release_notes_comments)
+        release_notes_comments = self.latest_release_notes.split('\n')
 
         if len(release_notes_comments) == 1:
-            if not (re.match(hyphen_release_notes_regex, self.release_notes) or
-                    re.match(one_line_release_notes_regex, self.release_notes)):
+            if not (re.match(hyphen_release_notes_regex, self.latest_release_notes) or
+                    re.match(one_line_release_notes_regex, self.latest_release_notes)):
                 print_error(F'File {self.release_notes_path} is not formatted according to '
-                            F'release notes standards.\nFix according to {release_notes_standard}')
+                            F'release notes standards.\nFix according to {self.RELEASE_NOTES_STANDARD}')
```

Comment:
I'd consider changing this to LINK_TO_RELEASE_NOTES_STANDARDS.
Had to go up and see what it was to understand